Sources/protoc-gen-swift/MessageGenerator.swift

```diff
@@ -67,6 +68,9 @@ class MessageGenerator {
       swiftRelativeName = sanitizeMessageTypeName(file.swiftPrefix + descriptor.name)
       swiftFullName = swiftRelativeName
     }
+    self.isAnyMessage = (isProto3 &&
+                         descriptor.name == "Any" &&
```

Comment:
Interesting; I would have used `file.package == ".google.protobuf"` as a condition here.  I suppose the filename check does the same thing, though.